coreapps-web-2.2/pom.xml

```diff
@@ -0,0 +1,84 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.openmrs.module</groupId>
+        <artifactId>coreapps</artifactId>
+        <version>1.32.0-SNAPSHOT</version>
```

Comment:
What is this change for, is it part of this commit

Reply:
@sherrif10 sure,@dkayiwa @ibacher   any suggestions on my above comment?

Reply:
@HerbertYiga  Have you tried adding functionalities  where neccessary in specified classes  and it fails without adding a submodule of 2.2, if that is i guess we should first create a ticket for adding a submodule of 2.2 then you proceed

Reply:
oh we can depend on version 2.0 as done[ here ](https://github.com/openmrs/openmrs-module-coreapps/blob/master/api-pre2.2/src/main/java/org/openmrs/module/coreapps/htmlformentry/EncounterDiagnosesElement.java)